packages/server/src/events/tasks.ts

```diff
@@ -29,7 +31,26 @@ export async function AddDislikeActivity(email: string, articleId: number) {}
  * Task ( Runs after the article save is created successfully ).
  * Add save activity to the logged in user
  */
-export async function AddSaveActivity(email: string, articleId: number) {}
+export async function AddSaveActivity(email: string, articleId: number) {
+    const prisma = PrismaClientSingleton.prisma;
+
+    const user = await prisma.user.findFirst({
+        where: {
+            email,
+        },
+    });
+
+    const newUserActivities = await prisma.userActivity.create({
+        data: {
+            action: 'save',
+            userId: user?.id!,
+            articleId,
+            updatedAt: new Date(),
```

Comment:
There is no need to provide the js date Prisma will take care of it and also it is optional field

Reply:
The `updatedAt` field has been created as a `required` so it's throwing run time error

Reply:
> The `updatedAt` field has been created as a `required` so it's throwing run time error

Yes, that was a bug. I have fixed it